config/test.exs

```diff
@@ -28,3 +28,20 @@ config :deep_thought, :deepl, auth_key: "auth_key"
 config :deep_thought, :slack,
   bot_token: "bot_token",
   signing_secret: "signing_secret"
+
+# Secret variables with dummy values
+deepl_auth_key = "auth_key"
+
+config :deep_thought, :deepl, auth_key: deepl_auth_key
+
+slack_bot_token = "bot_token"
```

Comment:
retrieve values from [these lines](https://github.com/monstar-lab-oss/deepl-for-slack-elixir/blob/master/.github/workflows/default.yml#L10-L15).